consensus/hotstuff/cruisectl/README.md

```diff
@@ -221,63 +222,77 @@ with parameters:
     
 The controller output $u[v]$ represents the amount of time by which the controller wishes to deviate from the ideal view duration $\tau_0$. In other words, the duration of view $v$ that the controller wants to set is
 ```math
 \widehat{\tau}[v] = \tau_0 - u[v]
 ```
 ---    
 
+### Limits of authority 
 
-For further details about 
-
-- the statistical model of the view duration, see [ID controller for ``block-rate-delay``](https://www.notion.so/ID-controller-for-block-rate-delay-cc9c2d9785ac4708a37bb952557b5ef4?pvs=21)
-- the simulation and controller tuning, see  [flow-internal/analyses/pacemaker_timing/2023-05_Blocktime_PID-controller](https://github.com/dapperlabs/flow-internal/tree/master/analyses/pacemaker_timing/2023-05_Blocktime_PID-controller) → [controller_tuning_v01.py](https://github.com/dapperlabs/flow-internal/blob/master/analyses/pacemaker_timing/2023-05_Blocktime_PID-controller/controller_tuning_v01.py)
-
-### Limits of authority
+[Latest update: Crescendo Upgrade, June 2024]
 
 In general, there is no bound on the output of the controller output $u$. However, it is important to limit the controller’s influence to keep $u$ within a sensible range.
 
 - upper bound on view duration $\widehat{\tau}[v]$ that we allow the controller to set:
   
-  The current timeout threshold is set to 2.5s. Therefore, the largest view duration we want to allow the  controller to set is 1.6s.
-  Thereby, approx. 900ms remain for message propagation, voting and constructing the child block, which will prevent the controller to drive the node into timeout with high probability. 
-    
+  The current timeout threshold is set to 1045ms and the largest view duration we want to allow the controller to set is $\tau_\textrm{max}$ = 910ms.
+  Thereby, we have a buffer $\beta$ = 135ms remaining for message propagation and the replicas validating the proposal for view $v$.
+
+  Note the subtle but important aspect: Primary for view $v$ controls duration of view $v-1$. This is because its proposal for view $v$
+  contains the proof (Quorum Certificate [QC]) that view $v-1$ concluded on the happy path. By observing the QC for view $v-1$, nodes enter the
+  subsequent view $v$.
+
+
 - lower bound on the view duration:
     
-  Let $t_\textnormal{p}[v]$ denote the time when the primary for view $v$ has constructed its block proposal. 
-  The time difference $t_\textnormal{p}[v] - t[v]$ between the primary entering the view and having its proposal
-  ready is the minimally required time to execute the protocol. The controller can only *delay* broadcasting the block,
+  Let $t_\textnormal{p}[v]$ denote the time when the primary for view $v$ has constructed its block proposal.
+  On the happy path, a replica concludes view $v-1$ and transitions to view $v$, when it observes the proposal for view $v$.
+  The time difference $t_\textnormal{p}[v] - t[v-1]$ between the primary observing the parent block (view $v-1$), collecting votes for it,
```

Comment:
equation in the previous version was erroneously using `t[v]` but it should be `t[v-1]`